src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -49,4 +49,20 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
    */
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] =
```

Comment:
@mcovarr I don't have a strong opinion on this one (yet). As an example, what would you call the `type`?